draft-iab-use-it-or-lose-it.md

```diff
@@ -292,15 +297,17 @@ but sometimes advantageous, despite some significant downsides.  Consequently,
 transparent proxies for cleartext HTTP were once commonplace.  Similarly, the
 DNS protocol was designed with intermediation in mind through its use of caching
 recursive resolvers {{?DNS=RFC1034}}.  What was less anticipated was the forced
-spoofing of DNS records by many middle-boxes such as those that inject
-authentication or pay-wall mechanisms as an authentication and authorization
-check, which are now prevalent in hotels, coffee shops and business networks.
-
-Middleboxes are also protocol participants, to the degree that they are able
-to observe and act in ways that affect the protocol.  The degree to which a
-middlebox participates varies from the basic functions that a router performs
-to full participation.  For example, a SIP back-to-back user agent (B2BUA)
-{{?B2BUA=RFC7092}} can be very deeply involved in the SIP protocol.
+spoofing of DNS records by middleboxes such as those that inject authentication
+or pay-wall mechanisms as an authentication and authorization check, which are
+now prevalent in hotels, coffee shops and business networks.
+
+Middleboxes are also protocol participants, to the degree that they are able to
+observe and act in ways that affect the protocol.  The degree to which a
+middlebox participates varies.  This can include middleboxes that apply their
```

Comment:
Cutting this sentence short ("The degree to which a middlebox participates varies.") makes it less clear in context. Can we extend it back to what it was?

Reply:
```suggestion
middlebox participates varies.  Middleboxes can also apply their
```